distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/config/UrlResourceReaderConfigurator.java

```diff
@@ -35,13 +36,29 @@ public UrlResourceReaderConfigurator(ConfigurationAdmin configAdmin) {
     this.configAdmin = configAdmin;
   }
 
+  @SuppressWarnings("squid:S2142")
   public void setUrlResourceReaderRootDirs(String... rootResourceDirs) throws IOException {
     Configuration configuration = configAdmin.getConfiguration(PID, null);
     Dictionary<String, Object> properties = new DictionaryMap<>();
     Set<String> rootResourceDirectories =
         ImmutableSet.<String>builder().add(rootResourceDirs).build();
     properties.put("rootResourceDirectories", rootResourceDirectories);
     configuration.update(properties);
+    for (int i = 0; i < 5; i++) {
```

Comment:
❓ Why `5`?

Reply:
Seemed like a nice round number and this is test code so I don't really care.